Example.md

```diff
@@ -0,0 +1,977 @@
+# A Step by Step Guide to Deployment Example
+
+Tekton deployments from GitHub
+
+## Requirements
+
+* Tekton Pipeline v0.10.1
+* Tekton Triggers v0.2.1
+* Set the following environment varialbes
+    * USER=\<OS user \>  (It should be automatically set by OS.)
+    * QUAY_USER=\<Ouay.io user\>
+    * GITHUB_USER=\<GitHub user\>
+    * DEPLOYMENT_PATH=deploy
+
+
+## Create Demo Git Repositories
+
+Login to Your GitHub account.  Create your demo Git repositories `taxi` and `taxi-stage-config`.  Then, import from the following repos, respectively.   
```

Comment:
Might want to explain how to import these?

Reply:
Github web UI offers import from existing repos as one of the options.   I'll add text to mention following Github option to import repos.